tests/testthat/test_dimension.R

```diff
@@ -0,0 +1,39 @@
+
+test_that("dimension_test1", {
+  config <- create.config(df = example.data, id = "sweden_at_eurovision_no_missing") %>%
+    add.dimension.column(
+      column.name = "Entrant",
+      label = "entrant",
+      description = "entrant",
+      from.existing = dimension.from.existing$age,
+      definition.uri = "https://www.youtube.com/watch?v=qCCpwFYibmU&list=PLSLc7UMdyVg590pc_gWh-d5ny0_PGC6Te&index=4",
+      cell.uri.template = "https://www.youtube.com/watch?v=qCCpwFYibmU&list=PLSLc7UMdyVg590pc_gWh-d5ny0_PGC6Te&index=4",
+      code.list = T,
+      from.template = "year"
+    )
+  json_string <- generate.json.configuration(config, file = "config.json")
+  validation_errors <- validate_qube_config(json_string)
+  expect_length(validation_errors, 0)
+})
+
+test_that("dimension_test2", {
+  config <- create.config(df = example.data, id = "sweden_at_eurovision_no_missing") %>%
+    add.dimension.column(
+      column.name = "Entrant",
+      label = "entrant",
+      description = "entrant",
+      from.existing = dimension.from.existing$age,
+      definition.uri = "https://www.youtube.com/watch?v=qCCpwFYibmU&list=PLSLc7UMdyVg590pc_gWh-d5ny0_PGC6Te&index=4",
```

Comment:
Can you remove these real links from here? I'd be uncomfortable merging these in given their content.